### DISCUSSION TITLE
[Support]: Is it possible to have 2 clients with the same Physical IP to be one the same network?

### DISCUSSION THREAD

**Discussion by ik9999** [sinamics/ztnet#253] [General]
### 📝 Describe the Problem You Are Having

Say i have 2 computers that have same public IP address, but cant connect to each other via LAN. Can i use ztnet to have them in the same network? For me, ztnet displays them as one client.

Is that a zerotier limitation?

### 🔖 Version

v0.5.5

### 💻 Operating System

Debian

**Comment by sinamics** [comment]
That should work just fine. 
Type `zerotier-cli info`  on each computer to see the node ID. This ID will show up in ztnet.

**Comment by ik9999** [comment]
`zerotier-cli info` shows different ids.
But in "Network Members" table, i only see one line. With the id of a user who connected first.

**Comment by ik9999** [comment]
Excuse, it was a problem on my end (i had a typo in network name).
